lib/board.rb

```diff
@@ -0,0 +1,48 @@
+require 'pry'
+
+class Board
+
+	attr_accessor :cells
+
+	def initialize
+		@cells = [' ',' ',' ',' ',' ',' ',' ',' ',' ']		
```

Comment:
I would just call `self.reset!` here and delegate to that method to create the default empty board.